dtests/src/test/java/io/snappydata/hydra/testDMLOps/partitionedRowExtn.inc

```diff
@@ -5,7 +5,7 @@ io.snappydata.hydra.testDMLOps.SnappySchemaPrms-snappyDDLExtn =
 " USING row OPTIONS(PERSISTENT '${persistenceMode}')"
 " USING row OPTIONS(PERSISTENT '${persistenceMode}')"
 " USING row OPTIONS(partition_by 'OrderID', buckets '13', redundancy '${redundantCopies}', PERSISTENT '${persistenceMode}')"
-" USING row OPTIONS(partition_by 'OrderID', buckets '13', COLOCATE_WITH 'orders', redundancy '${redundantCopies}', PERSISTENT '${persistenceMode}')"
+" USING row OPTIONS(partition_by 'OrderID', buckets '13', COLOCATE_WITH 'gemfire1.orders', redundancy '${redundantCopies}', PERSISTENT '${persistenceMode}')"
```

Comment:
Remove the schema name, as tests without security enabled will fail when they dont find 'gemfire1' schema.